renderer/utils/format-time.js

```diff
@@ -6,19 +6,17 @@ const formatTime = (time, options) => {
     ...options
   };
 
-  const format = `m:ss${options.showMilliseconds ? '.SS' : ''}`;
-
   const durationFormatted = options.extra ?
-    `  (${moment()
-      .startOf('day')
-      .milliseconds(options.extra * 1000)
-      .format(format)})` :
+    `  (${format(options.extra, options)})` :
     '';
 
-  return `${moment()
-    .startOf('day')
-    .milliseconds(time * 1000)
-    .format(format)}${durationFormatted}`;
+  return `${format(time, options)}${durationFormatted}`;
+};
+
+const format = (time, {showMilliseconds} = {}) => {
+  const formatString = `${time >= 60 * 60 ? 'h:m' : ''}m:ss${showMilliseconds ? '.SS' : ''}`;
```

Comment:
Not related to this PR, but I think we should show it as
```suggestion
  const formatString = `${time >= 60 * 60 ? 'hh:mm' : ''}mm:ss${showMilliseconds ? '.SS' : ''}`;
```

This is how most video players show duration.

Reply:
I think you meant this:

```suggestion
  const formatString = `${time >= 60 * 60 ? 'h:' : ''}mm:ss${showMilliseconds ? '.SS' : ''}`;
```

right? Otherwise if it's more than an hour the format becomes `hh:mmmm:ss`

Reply:
I meant:

```suggestion
  const formatString = `${time >= 60 * 60 ? 'hh:m' : ''}m:ss${showMilliseconds ? '.SS' : ''}`;
```